CMakeLists.txt

```diff
@@ -86,6 +86,7 @@ option(ENABLE_DOCS "Indicates whether documentation should be built." ON)
 option(ENABLE_TUTORIALS "Indicates whether tutorials should be build. Note that this also depends on the availability of (pdf)latex." ON)
 option(WITH_GUI "Build GUI parts of OpenMS (TOPPView&Co). This requires QtGui." ON)
 option(NO_WEBENGINE_WIDGETS "Do not use QtWebengineWidgets. Disables Javascript views in TOPPView." OFF)
+option(ENABLE_TORCH "Enable pyTorch" OFF)
```

Comment:
```suggestion
option(ENABLE_OPENMS_ML "Enable machine learning library to interface with e.g. pyTorch." OFF)
```

Reply:
is the check for ENABLE_OPENMS_ML put in [src/openms_ml/CMakeLists.txt](https://github.com/OpenMS/OpenMS/pull/6541/files/66d987c2abaf02d87f712c78bd193cbffb999b7f#diff-b2c6494e225e03dc969755b7bffc33a5388a89a7066f318e7844bdf5e0be1c37)
? 
or already when add_subdirectory adds the CMakeLists ?

Reply:
In the top-level. Then only add subdirectory if enabled.